app/test_main.py

```diff
@@ -34,3 +36,25 @@ def test_get_human_age(
     expected_human_age: list[int, int],
 ) -> None:
     assert get_human_age(cat_age, dog_age) == expected_human_age
+
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected_error",
+    [
+        ("14", 14, TypeError),
+        (24, "24", TypeError),
+        ("28", "29", TypeError),
+    ],
+    ids=[
+        "Cat age should be Integer, not String",
+        "Dog age should be Integer, not String",
+        "Cat age and Dog age should be Integers, not Strings",
+    ],
+)
+def test_get_human_age_invalid_input(
```

Comment:
```suggestion
def test_get_human_age_invalid_input_type
```
It's ok name, but as for now it checks only type, so better to explicit mention it.